From f156f8da25f681b7ecf51580ee2aa19122cd45c2 Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Fri, 23 Sep 2022 15:18:36 +0200
Subject: [PATCH 0336/1695] drivers: stm32_exti: add generic compatible
 'st,stm32mp1-exti'

Moving the interrupt routing table in DT property 'interrupt-map'
makes useless differentiating the instance through the compatible
string.
Add the generic compatible 'st,stm32mp1-exti' in the struct
dt_device_match.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git dacefab92e45acccf023e3a03062b11bc51b082c drivers: stm32_exti: add generic compatible 'st,stm32mp1-exti']
Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Change-Id: Ie4089b4bbc8cad5872014abb54e0a3507bb3fed5
---
 core/drivers/stm32_exti.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/drivers/stm32_exti.c b/core/drivers/stm32_exti.c
index c75246167..b3f701734 100644
--- a/core/drivers/stm32_exti.c
+++ b/core/drivers/stm32_exti.c
@@ -363,6 +363,7 @@ static TEE_Result stm32_exti_probe(const void *fdt, int node,
 }
 
 static const struct dt_device_match stm32_exti_match_table[] = {
+	{ .compatible = "st,stm32mp1-exti" },
 	{ .compatible = "st,stm32mp13-exti" },
 	{ }
 };
-- 
2.47.2

